llvm/test/TableGen/riscv-target-def.td

```diff
@@ -12,6 +12,11 @@ class RISCVExtension<string name, int major, int minor, string desc,
   bit Experimental = false;
 }
 
+class RISCVExtensionBitmask<bits<3> groupID, int bitmaskShift> {
+    bits<3> GroupID = groupID;
```

Comment:
Can we use `int` for these if we store bit position instead of bit mask?

Reply:
Both use `int` now.